gitops/azure-devops/build.sh

```diff
@@ -175,7 +179,8 @@ function verify() {
 
 # Run functions
 function verify_and_push() {
```

Comment:
I suppose we could be more explicit here to. Maybe "verify_pull_request_and_merge" Thoughts?